platform-core-module/src/test/java/com/softicar/platform/core/module/program/AbstractProgramTest.java

```diff
@@ -0,0 +1,25 @@
+package com.softicar.platform.core.module.program;
+
+import com.softicar.platform.core.module.user.AGUser;
+import com.softicar.platform.db.runtime.test.AbstractDbTest;
+
+public abstract class AbstractProgramTest extends AbstractDbTest {
+
+	protected final AGUser user;
+
+	public AbstractProgramTest() {
+
+		this.user = insertUser();
+	}
+
+	private AGUser insertUser() {
+
+		return new AGUser()//
+			.setActive(false)
```

Comment:
Why would we insert an _inactive_ user? This could (or even should) cause all sorts of problems.